index.html

```diff
@@ -180,11 +202,16 @@
             document.getElementById("teal-options").appendChild(optionDiv);
         }
 
+        // var el = document.getElementById('navbar');
```

Comment:
Please attach a screenshot/gif of the UI functionality you are introducing. In addition, it seems like the navigation link is not functional (the relevant JavaScript seems to be commented out).